Python/Product/Analysis/LanguageServer/Messages.cs

```diff
@@ -63,7 +63,7 @@ public sealed class ShowMessageEventArgs : EventArgs {
     }
 
     [Serializable]
-    public struct ShowMessageRequestParams {
+    public class ShowMessageRequestParams {
```

Comment:
You have converted _some_ of the structs into classes. Does it matter only for those several types?

Reply:
Those which are important are the JSON serialization. Others can remain structs.